packages/web/src/scss/components/Header/_Header.scss

```diff
@@ -263,8 +259,8 @@ button.Header__link--current {
 
 // stylelint-disable selector-max-class -- Allow targeting interaction classes.
 .Header__bar.is-open::after,
-.Header__bar.is-open > .Header__backdrop,
+.Header__bar.is-open::backdrop,
 .Header__dialog.is-open::after,
-.Header__dialog.is-open > .Header__backdrop {
+.Header__dialog.is-open::backdrop {
```

Comment:
we might change this to `[open]` selector (as in Modal) because it is natively there.

Reply:
Filed as a topic for Thursday.